README.md

```diff
@@ -5,7 +5,37 @@ You might not care when launching a single container on your laptop, but in the
 
 ## The problem
 
-TBD.
+### Too many containers require privileges to run.
```

Comment:
Yup, SGTM @cgwalters … @rhatdan you plan to add that or shall I merge and add it myself?